src/common/components/UserForm/index.jsx

```diff
@@ -38,6 +88,57 @@ class UserForm extends Component {
     const submitDisabled = Boolean(pristine || submitting || invalid)
     const title = `Edit User: ${user.handle}`
 
+    const canBeDeactivated = user.active && userCan(currentUser, 'deactivateUser')
+    const canBeReactivated = !user.active && userCan(currentUser, 'reactivateUser')
```

Comment:
It would be preferable to keep this logic in the container; ideally, simple presentational components don't have to be concerned with authorization matters.

We can serve the same UX with a slightly cleaner implementation by determining the values of `canBeDeactivated` and `canBeReactivated` up in the parent container and passing them down as props.